justifi-tech/web-component-library
# FE-183: Handle errors for arrays of fields

CHANGES:

- `businessFormSchema.ts`: Extracted Representative and Address schema, and created the owner schema;
- Added the BusinessOwners sub form to  BusinessForm component;
- `business-address.spec.tsx`: Fix `defaultvalue` prop case;
- `business-owners.tsx`: Created the business owners form;
-  `business-address-form.scss`, `business-form.scss`, `business-generic-info.scss`, `business-owners.scss`, `business-representative.scss`: Added CSS variables support;
- `business-form.stories.tsx`: Added a story for the `<justifi-business-form />` component;

<img width="1653" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/22480988/28cdc83f-3cde-4bdd-98cf-a5b65041af6d">




Links
-----



Developer considerations
--------------

  - If this is a MAJOR version, what are the breaking API changes? Was anything added to the API?
  - If this is a MINOR version, what feature was added? Why would consumers want this new version?
  - If this is a PATCH version, what was fixed?


Steps for Testing/QA
--------------------

- Go to http://localhost:6006/?path=/story/components-business-form--basic 
- Should be possible to see the BusinessForm component with subforms
